src/main/java/de/blackforestsolutions/datamodel/TravelPoint.java

```diff
@@ -42,9 +43,35 @@
 
     private String terminal = "";
 
+    private Date departureTime;
+
+    private Date arrivalTime;
+
     public TravelPoint() {
     }
 
+    public Date getDepartureTime() {
```

Comment:
Kleine Interessensgeschichte am Rande weisst du warum wir Date Clonen müssen in Gettern und Settern?